examples/django_example/myapp/__init__.py

```diff
@@ -0,0 +1,16 @@
+import os  
+import signal  
+import sys 
+from .langfuse_integration import langfuse_flush
+
+def shutdwon_handler(*args):  
+    """
+    This function handles the shutdown process.
+    
+    It calls the langfuse_flush function to flush any pending changes,
+    and then exits the program with a status code of 0.
+    """
+    langfuse_flush()
+    sys.exit(0)
+
+signal.signal(signal.SIGINT, shutdwon_handler)  
```

Comment:
I think we can also listen to sigterm here as this is used to gracefully terminate a process. https://www.baeldung.com/linux/sigint-and-other-termination-signals